src/ic-websocket-gateway/src/client_session_handler.rs

```diff
@@ -180,12 +185,28 @@ impl ClientSessionHandler {
                 Ok(Some(IcWsSessionState::Open)) => {
                     client_session_span.in_scope(|| {
                         debug!("Client session opened");
+
+                        gauge!("clients_connected").increment(1.0);
```

Comment:
@AvoidMelon can you also add the label here (and wherever else necessary) as we did in our last call? Just as reference for future implementation